state/indexer/block/kv/kv_test.go

```diff
@@ -511,56 +512,6 @@ func TestBigInt(t *testing.T) {
 	}
 }
 
-func TestKeyArray(t *testing.T) {
```

Comment:
Do you have to delete this test with the renaming? Can you move all the unused functions from `util` back into this file?